src/fu-security-attr.c

```diff
@@ -125,14 +125,14 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
 		/* TRANSLATORS: Title: TPM = Trusted Platform Module */
 		return g_strdup(_("TPM v2.0"));
 	}
-	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MEI_MANUFACTURING_MODE) == 0) {
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_MANUFACTURING_MODE) == 0) {
 		const gchar *kind = fwupd_security_attr_get_metadata(attr, "kind");
 		if (kind != NULL) {
 			/* TRANSLATORS: Title: %s is ME kind, e.g. CSME/TXT */
 			return g_strdup_printf(_("%s manufacturing mode"), kind);
 		}
-		/* TRANSLATORS: Title: MEI = Intel Management Engine */
-		return g_strdup(_("MEI manufacturing mode"));
+		/* TRANSLATORS: Title: A SOC mode with less permissive security */
+		return g_strdup(_("Manufacturing/Debug mode"));
```

Comment:
I'm not sure about this. I don't think we should combine things that are not 100% the same. I actually think it makes sense to have a `MEI manufacturing mode` on Intel that points to documentation about Intel AMT and CSME, and a `PSP Debug Mode` item on AMD that points to information about AMD PSP. I think it's fine to combine items like IOMMU where the implementation is basically the same, and the security problems with having it disabled are exactly the same between vendors - but I don't think we should collapse things that are actually quite different into one property.

Reply:
My goal is to avoid too many attributes that are manufacturer specific concepts.

Technically the PSP implementation is tying two things in:

1. part is fused
2. part is locked from debug

I am tempted to still combine the first with a generic "manufacturing mode" and make a new option for debug locked.

Reply:
> avoid too many attributes that are manufacturer specific

I think I'd rather have dozens of vendor specific attributes rather than trying to squash different things together and trying to find a suitable HSI level integer for the superset.

Reply:
Adjusted